src/main/scala/sbtlicensereport/license/LicenseInfo.scala

```diff
@@ -57,4 +57,9 @@ object LicenseInfo {
   val MPL = LicenseInfo(LicenseCategory.Mozilla, "Mozilla Public License 2.0", "https://www.mozilla.org/MPL/2.0/")
   val BouncyCastle =
     LicenseInfo(LicenseCategory.BouncyCastle, "Bouncy Castle Licence", "https://www.bouncycastle.org/license.html")
+  val IBM = LicenseInfo(
+    LicenseCategory.IBM,
+    "IBM International Program License Agreement",
+    "https://www14.software.ibm.com/cgi-bin/weblap/lap.pl?li_formnum=L-APIG-C2SEH5"
```

Comment:
This URL references a page that contains a textbox that says

> The Programs listed below are licensed under the following License Information terms and conditions in addition to the Program license terms previously agreed to by Client and IBM. If Client does not have previously agreed to license terms in effect for the Program, the International License Agreement for Evaluation of Programs (Z125-5543-05) applies.
>
> Program Name (Program Number):
>
> IBM MQ 9.2 - 07/2021 (Evaluation)

In other words, not only is this license specific to a single program, it's a license URL specific to a single version. I don't know how sbt-license-info should handle it, but using the name `val IBM` or including this in the plugin in general seems wrong here.

Reply:
Yeah let me look at this in more detail, its brought in by IBM MQ binary (which is used for testing) and there is an argument it should just be handled in the project and not in sbt-license-report

Reply:
Ideally we could add a category called "IBM International Program License Agreement" or `IBM_IPLA`, and licenses with the name gets categorized into it.

Reply:
Okay I did manage to find a generic license, so I will re-open PR.